structured-geocoding.md

```diff
@@ -0,0 +1,143 @@
+# Structured geocoding
+
+With structured geocoding, you can search for the individual parts of a location. Structured geocoding is an option on the [`search` endpoint](search.md), where a query takes the form of https://search.mapzen.com/v1/search/structured.
+
+For example, you want to find `30 West 26th Street, New York, NY`. With the geocoding parameter for `search`, you can only enter the entire location as one string, such as `text=30 West 26th Street, New York, NY`. However, with `search\structured`, you can specify that this location is composed of a street address, a locality, and a region.
+
+```
+{
+  address: '30 West 26th Street',
+  locality: 'New York',
+  region: 'NY'
+}
+```
+
+Structured geocoding can improve how the items in your query are parsed and interpreted in a search. An address such as `10 Park Place North Charleston South Carolina` could be viewed as a city name containing a directional (North Charleston in South Carolina), or as a street with a post-directional (10 Park Place North). By separating the components of the search input, you are reducing ambiguity in your query. This is also helpful because addresses and postal codes around the world are often formatted and ordered differently.
+
+One common use for structured geocoding is for geocoding a list or table of addresses, where each column represents a different portion of a location.
+
+| address | city | state | country |
+| ------- | ---- | ----- | ------- |
+| 1600 Pennsylvania Ave | Washington | DC | US |
+| 10 Downing Street | London | | GB |
+| 55 Rue du Faubourg Saint-Honoré | Paris | | FR |
+| Bulevardul Geniului 1 | Bucharest | | Romania |
+
+With a regular `search` query, you would need to concatenate these columns into one string, but `search/structured` allows you to define a query that maintains the individual fields for address, city, state, and country.
+
+## Structured geocoding parameters
+
+You can use structured geocoding to search for the following parameters:
+
+* address
+* neighbourhood
+* borough
+* locality
+* county
+* region
+* postalcode
+* country
+
+Note that the other [`search` parameters](search.md/#available-search-parameters) can also be combined with these, allowing you to filter and prioritize your results.
+
+### address
+
+The `address` parameter can contain a full address with house number or only a street name.
+
+_Examples_
+
+* [201 Spear Street](http://search.mapzen.com/v1/search/structured?address=201%20Spear%20Street&locality=San%20Francisco&region=CA)
```

Comment:
It's really annoying, but for some reason clicking a URL containing spaces or `%20`s doesn't work, they have to be `+`s.